Sources/Core/DTOLoader.swift

```diff
@@ -0,0 +1,25 @@
+//  DTOLoader.swift
+
+import Foundation
+import Yams
+
+struct DTOLoader {
+
+    enum GeneratorError: Error {
+        case invalidFormat(String)
+    }
+
+    func loadDto<T: Decodable>(url: URL) throws -> T {
```

Comment:
```suggestion
    func loadDTO<T: Decodable>(url: URL) throws -> T {
```

Reply:
I was on a rampage about casing for acronyms in the past but some people convinced me that uppercasing the whole acronym in method names is such a boomer thing to do and I should just capitalise the word 🤷 
I'm with you, I'll rename it.